app/moves/middleware.js

```diff
@@ -116,24 +115,46 @@ const movesMiddleware = {
           return moveService
             .getMovesCount({ dateRange, status: moveType.filter, locationId })
             .then(count => {
-              const { label } = moveType
               return {
-                label,
+                ...moveType,
                 value: count,
                 active: moveType.filter === req.params.status,
                 href: `${req.baseUrl}/${period}/${date}${
                   locationId ? '/' + locationId : ''
                 }/${moveType.filter}`,
               }
             })
-            .then(presenters.moveTypesToFilterComponent)
         })
       )
+      res.locals.moveTypeNavigation = res.locals.moveTypeNavigation.map(
+        presenters.moveTypesToFilterComponent
+      )
       next()
     } catch (error) {
       next(error)
     }
   },
+  setDashboardMoveSummary: (req, res, next) => {
+    let totalMoves = {
+      label: 'moves::dashboard.filter.total',
+      filter: 'total',
+      href: find(res.locals.moveTypeNavigation, { filter: 'proposed' }).href,
+    }
+    totalMoves.value = res.locals.moveTypeNavigation.reduce(
+      (accumulator, moveType) => {
+        return (accumulator += moveType.value)
+      },
+      0
+    )
+    totalMoves = presenters.moveTypesToFilterComponent(totalMoves)
+    res.locals.dashboardMoveSummary = [
+      { ...totalMoves },
```

Comment:
That was my bad in the original suggestion, you can drop the spread and then you don't need to call inside in an object: 

```javascript
res.locals.dashboardMoveSummary = [
  totalMoves,
  ...reject(res.locals.moveTypeNavigation, {
    filter: 'proposed',
  }),
]
```